dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -176,7 +204,38 @@ def _AwaitFlow(self, client, flow_id):
       if status.state == flows_pb2.FlowContext.TERMINATED:
         self.logger.info('{0:s}: Complete'.format(flow_id))
         break
+
       time.sleep(self._CHECK_FLOW_INTERVAL_SEC)
+      if not self.skip_offline_clients:
+        continue
+
+      client_last_seen = datetime.datetime.utcfromtimestamp(
+          client.data.last_seen_at / 1000000)
+      now = datetime.datetime.utcnow()
+      if (now - client_last_seen).total_seconds() > self._MAX_OFFLINE_TIME_SEC:
+        self.logger.warning(
+              'Client {0:s} has been offline for more than an'
+              ' hour, skipping...'.format(
```

Comment:
well, that's not necessarily true... it's been more than MAX_OFFLINE_TIME_SEC since the host was still last seen....

so .... if I change the offline time sec variable, or we later change to be configurable, then this will not hold... maybe have this rather as `for more than {self._MAX_OFFLINE_TIME_SEC} seconds` or calculate the minutes `f'for more than {self._MAX_OFFLINE_TIME_SEC / 60} minutes'` or something like that

Reply:
Good point.